tests/test_zendesk_client.py

```diff
@@ -168,3 +169,123 @@ def test_zendesk_client_send_ticket_data_to_zendesk_error(zendesk_client, app, r
         zendesk_client.send_ticket_data_to_zendesk({'ticket_data': 'Ticket content'})
 
     mock_logger.assert_called_with("Zendesk create ticket request failed with 401 '{'foo': 'bar'}'")
+
+
+@pytest.mark.parametrize('p1_arg, expected_tags, expected_priority', (
+    (
+        {},
+        ['govuk_notify_support'],
+        'normal',
+    ),
+    (
+        {
+            'p1': False,
+        },
+        ['govuk_notify_support'],
+        'normal',
+    ),
+    (
+        {
+            'p1': True,
+        },
+        ['govuk_notify_emergency'],
+        'urgent',
+    ),
+))
+def test_notify_support_ticket_request_data(p1_arg, expected_tags, expected_priority):
+    notify_ticket_form = NotifySupportTicket('subject', 'message', 'ticket_type', **p1_arg)
```

Comment:
I've changed everywhere that we were using the string of 'ticket_type' in the tests to one of the real values.

If you do send data to Zendesk and it has an unexpected value as the ticket_type then the ticket will still be created - a bad value won't cause an error. Zendesk also silently ignores any tag values in the data that it doesn't know about.

I think validation is worth considering, even though bad data doesn't really do anything, but this might be one to come back to when all ticket types use the new class and the code has been simplified.